extensions/azurecore/src/extension.ts

```diff
@@ -81,6 +81,12 @@ export async function activate(context: vscode.ExtensionContext): Promise<azurec
 	if (!storagePath) {
 		return undefined;
 	}
+
+	const config = vscode.workspace.getConfiguration('accounts.azure.auth');
+	if (vscode.env.uiKind === vscode.UIKind.Web) {
```

Comment:
This is going to make it so that these are reset every time the user launches, which doesn't seem ideal. 

What's the intention of this change here?

Reply:
Intention is to have adsweb with different default settings which makes sense for web mode. Like device code auth set to true by default for web mode.